_invoke

```diff
@@ -3,25 +3,40 @@
 typeset -A opt_args
 local context state line
 
-_invoke_subcommand () {
-  if [ -f tasks.py ]; then
-    local cache_policy
+function _invoke_subcommands {
+    COMMAND_COMPLETIONS=( $(invoke -l |tr -d ' ' | tail -n+2 | sed '$ d' | tr '\n' ' ') )
```

Comment:
testing my [tasks.py](https://github.com/hhatto/otamapy/blob/master/tasks.py), result would be as follows.

```
 buildbuildingaotamapymodule cleancleandevelopmentenvironment examplerunexamplecode installinstallotamapy install_libotamainstalltolibotama pypiregregstertoPyPI testrununittest
```
Can you modify because the python docstring is included?